openhtf/io/frontend/__main__.py

```diff
@@ -174,7 +165,8 @@ def GetStationState(self, host, port):
       _LOG.warning(
           'Store was queried for an unknown station: %s:%s' % (host, port))
       return None
-    response = self.MessageFramework(host, port)
+    self.stations[host, port].Query()
```

Comment:
this is going to break in all sorts of exciting ways if you give it a host/port of an unknown station (it'll try to use the default values of None, None for host/port to build the request address of `http://None:None` i think).